docs/getting-started.md

```diff
@@ -39,6 +30,38 @@ podman run -i --rm quay.io/coreos/fcct:release --pretty --strict < your_config.f
 podman run --rm -v /path/to/your_config.fcc:/config.fcc:z quay.io/coreos/fcct:release --pretty --strict /config.fcc > transpiled_config.ign
 ```
 
+You may also setup the following alias in your shell configuration:
```

Comment:
`setup` is a noun; the verb is`set up`.  But `create` might be better.

Reply:
Went with 'add'